actors/market/src/ext.rs

```diff
@@ -20,6 +20,15 @@ pub mod account {
         #[serde(with = "serde_bytes")]
         pub message: Vec<u8>,
     }
+
+    pub const MARKET_NOTIFY_DEAL: u64 = frc42_dispatch::method_hash!("MarketNotifyDeal");
+
+    #[derive(Serialize_tuple, Deserialize_tuple)]
+    pub struct MarketNotifyDealParams {
+        #[serde(with = "serde_bytes")]
+        pub proposal: Vec<u8>,
+        pub deal_id: u64,
+    }
```

Comment:
I don't think `ext` is the right place for this, since the primary definition is this actor. Other actors would have it in ext::market. This is a fairly minor thing tho.